common/simd/arm/sse2neon.h

```diff
@@ -1731,10 +1731,12 @@ FORCE_INLINE __m128 _mm_div_ss(__m128 a, __m128 b)
 
 // Free aligned memory that was allocated with _mm_malloc.
 // https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_free
+#if !(defined _WIN32 && defined __has_include && __has_include(<mm_malloc.h>))
```

Comment:
Clang on MinGW defines `_mm_free` in `mm_malloc.h`, which is included by `malloc.h` and in turn `stdlib.h`.

Reply:
Looks like this is addressed by sse2neon upstream: https://github.com/DLTcollab/sse2neon/blob/7bd15eac51e36bf7426052f8515358cb665d8c04/sse2neon.h#L107, consider upgrading.